crates/sequencer/src/rpc.rs

```diff
@@ -80,17 +82,21 @@ pub(crate) fn create_rpc_module<C: sov_modules_api::Context>(
             Ok::<(), ErrorObjectOwned>(())
         })?;
 
-        rpc.register_async_method("da_publishBlock", |_, _ctx| async move {
-            info!("Sequencer: da_publishBlock");
-            let da = MockDaService::new(MockAddress::from([0; 32]));
-            da.publish_test_block().await.map_err(|e| {
-                ErrorObjectOwned::owned(
-                    INTERNAL_ERROR_CODE,
-                    INTERNAL_ERROR_MSG,
-                    Some(format!("Could not publish mock-da block: {e}")),
-                )
-            })?;
-            Ok::<(), ErrorObjectOwned>(())
+        let da_db_path = da_db_path.expect("DA DB should be set");
```

Comment:
Put inside context?

Reply:
We want to panic in the initialization phase if db path is not specified. IMO, better than panicing when the handler is called.